src/Config/InfectionConfig.php

```diff
@@ -167,4 +172,14 @@ public function getTestFramework(): string
     {
         return $this->config->testFramework ?? 'phpunit';
     }
+
+    public function getInitialTestsPhpOptions(): string
```

Comment:
How about moving these new options to a separate PR?
It would reduce the size of this PR a little bit, and make it easier to review.

And could you also add tests for these new methods?